src/test/java/org/fogbowcloud/saps/engine/core/archiver/storage/SwiftPermanentStorageTest.java

```diff
@@ -269,80 +269,78 @@ public void failureTestWhenTryingToArchiveFailureTaskWithArchiverInDebugMode() t
     }
 
     @Test
-    public void testToDeleteSuccessfulTaskWithoutArchiverInDebugMode()
-        throws IOException, PermanentStorageException {
+    public void testToDeleteSuccessfulTaskWithoutArchiverInDebugMode() throws Exception {
         SwiftAPIClient swiftAPIClient = mock(SwiftAPIClient.class);
         Properties properties = createDefaultPropertiesWithoutArchiverInDebugMode();
         PermanentStorage permanentStorage = new SwiftPermanentStorage(properties, swiftAPIClient);
 
-        Mockito.when(swiftAPIClient.listFilesWithPrefix(Mockito.eq(MOCK_CONTAINER_NAME), Mockito.eq(MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task01))).thenReturn(filesTask01);
+        Mockito.when(swiftAPIClient.listFiles(Mockito.eq(MOCK_CONTAINER_NAME), Mockito.eq(MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task01))).thenReturn(filesTask01);
         Mockito.doAnswer((i) -> {
             return null;
         }).when(swiftAPIClient).deleteFile(Mockito.eq(MOCK_CONTAINER_NAME), Mockito.argThat(new StartsWith(MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task01)));
 
-        boolean deleteTask01 = permanentStorage.delete(task01);
+        boolean deleteTask01 = permanentStorage.archive(task01);
```

Comment:
Change to `permanentStorage.delete(task01)`.

Reply:
Done